### PR TITLE
Add cleanup script - closes #747

### LINKED ISSUE #747: Add clean scripts
### Expected behavior

There should be an easy way to remove fragile files from a package, e.g. node_modules and distribution directories, and an easy way to remove them from all packages at once from the root.

### Actual behavior

We have no such scripts, but @yatki has some locally.

### Which version(s) does this affect? (Environment, OS, etc...)

1.1.0

### PR DESCRIPTION
**Pull request by yatki**
### Review checklist

* The PR resolves #747 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the
	[commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: scripts/cleanup.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Please run this script only in the project root folder.
+#
+# Usage
+# ./scripts/cleanup.sh
```
Comment (willclarktech) [inline scripts/cleanup.sh]: Ideally I'd like a `npm run clean` script in each package, and a `"clean": "lerna run clean"` script in the root.

File: scripts/cleanup.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+#
+# To delete node_modules folder:
+# ./scripts/cleanup.sh all
+
+rm -rf packages/**/dist/
```
Comment (willclarktech) [inline scripts/cleanup.sh]: Why is the `-f` flag necessary for any of these?

File: scripts/cleanup.sh [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+rm -rf packages/**/dist-browser/
+rm -rf packages/**/browsertest.build/
+rm -rf packages/**/.nyc_output/
+rm -rf packages/**/*.log
+
+echo "Distribution, build and log files were deleted from all packages!"
+
+if [ "$1" == "all" ]
+	then
+		rm -rf packages/**/node_modules/;
```
Comment (willclarktech) [inline scripts/cleanup.sh]: The root package also has `node_modules`.

File: scripts/cleanup.sh [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+# ./scripts/cleanup.sh all
+
+rm -rf packages/**/dist/
+rm -rf packages/**/dist-node/
+rm -rf packages/**/dist-browser/
+rm -rf packages/**/browsertest.build/
+rm -rf packages/**/.nyc_output/
+rm -rf packages/**/*.log
+
+echo "Distribution, build and log files were deleted from all packages!"
```
Comment (willclarktech) [inline scripts/cleanup.sh]: We probably don't need feedback in the success case.

**Comment by yatki** [comment]
No need for this pr anymore since we decided to have a clean up function for reach package. @willclarktech

**Pull request closed without merging**